Source/Core/Core/HW/EXI/EXI_DeviceEthernetTAP_Unix.cpp

```diff
@@ -28,7 +28,22 @@ namespace ExpansionInterface
 #define NOTIMPLEMENTED(Name)                                                                       \
   NOTICE_LOG(SP1, "CEXIETHERNET::%s not implemented for your UNIX", Name);
```

Comment:
This macro is now out of date after your restructuring.
```suggestion
   NOTICE_LOG(SP1, "%s not implemented for your UNIX", __PRETTY_FUNCTION__)
```
Should work with GCC, Clang and android build tools. Don't forget to drop the argument from the macro and fix all uses of it.

Reply:
AFAIK Visual Studio doesn't have `__PRETTY_FUNCTION__` but uses `__FUNCTION__` instead. But I guess since this is *nix specific code, it should be fine...?

Reply:
I figured it would be fine since there is a Win32 file already. If it's still undesired, just disregard this suggestion.